interfaces/vrep/robots/YouBotVrepRobot.m

```diff
@@ -17,8 +17,8 @@
 %       will become "youBot#0", a third robot, "youBot#1", and so on.
 %
 %   YouBotVrepRobot Methods:
-%       send_q_to_vrep - Sends the joint configurations to VREP
```

Comment:
Careful removing methods, we need at least one version of backwards compatibility

Reply:
Backwards compatibility added in [a02ad1b](https://github.com/dqrobotics/matlab/pull/108/commits/a02ad1ba01016269077e04b6465c43d803049bd1).

Reply:
In the review view, this is still deleted and not added anywhere else. Maybe I'm doing something wrong?

Reply:
In [a02ad1b](https://github.com/dqrobotics/matlab/pull/108/commits/a02ad1ba01016269077e04b6465c43d803049bd1) I added backwards compatibility to both `send_q_to_vrep` and `get_q_from_vrep` to the class `DQ_VrepRobot`, where they used to be abstract methods.

``` matlab
function send_q_to_vrep(obj, q)
   % For backwards compatibility only. Do not use this method.

   warning('Deprecated. Use set_configuration_space_positions() instead.')
   obj.set_configuration_space_positions(q)
end

function q = get_q_from_vrep(obj)
   % For backwards compatibility only. Do not use this method.

   warning('Deprecated. Use get_configuration_space_positions() instead.')
   q = obj.get_configuration_space_positions();
end
```

As such, the use of something similar to

``` matlab
vi = DQ_VrepInterface();
vi.connect('127.0.0.1',19997);
vrep_robot = YouBotVrepRobot("youBot", vi);
vrep_robot.get_q_from_vrep
```

yields the result
``` matlab
Warning: Deprecated. Use get_configuration_space_positions() instead. 
> In DQ_VrepRobot/get_q_from_vrep (line 65) 

ans =

    1.2068
   -1.8492
    3.6463
    0.0001
    0.5395
    0.9149
    1.2685
   -0.0001
```

That's the desired result, right? The deprecated method `send_q_to_vrep` works similarly.

Reply:
Hi, @ffasilva ,

The desired result is to leave the original code unchanged as much as possible and work the new code around it. 

I understand why you chose to do it this way, but I'll condition the acceptance of this to these examples working without modification:
https://github.com/dqrobotics/matlab-examples/tree/master/vrep

Reply:
Hi @mmmarinho,

I see what you mean, but I figured that deprecating the methods in all the concrete classes that inherit from `DQ_VrepRobot`, rather than just doing it directly on the abstract class itself, would have generated even more modifications to the original code.

As for your request, the deprecation of the methods is working as intended. The [simulation-ram-paper](https://github.com/dqrobotics/matlab-examples/tree/master/vrep/simulation-ram-paper) runs without any problems:

![simulation_ram_paper](https://github.com/user-attachments/assets/51f4c3b5-d028-4a13-8eff-78716e80f95a)

However, the [constrained_kinematic_control](https://github.com/dqrobotics/matlab-examples/tree/master/vrep/constrained_kinematic_control) example don't use the VrepRobot classes at all. Which means it obviously works, but it also doesn't help us validate my modifications. I raised an [issue](https://github.com/dqrobotics/matlab-examples/issues/11) to update this example in the future.

Kind regards,
Frederico

Reply:
Thanks, @ffasilva. Your reasoning is sensible as you have minimized the number of modifications, and the example still runs. Therefore, your solution is acceptable.